scripts/default_feeds.sh

```diff
@@ -1 +1,2 @@
-DEFAULT_FEEDS="$(awk '{print $2}' openwrt/feeds.conf.default)"
+# only list feeds which don't start with #
+DEFAULT_FEEDS="$(grep -v ^# openwrt/feeds.conf.default | awk '{print $2}')"
```

Comment:
Combining grep and awk in a single line is rarely useful. Shorter:
```sh
awk '!/^#/ {print $2}' openwrt/feeds.conf.default
```
Also, remove the word "only" from the comment.

Reply:
@NeoRaider  your awk skills are better than mine ... :-)

changed as suggested